LICENSE.txt

```diff
@@ -1,6 +1,8 @@
 Bitfusion with Kubernetes Integration
 Copyright 2020 VMware, Inc.
 
+NOTE:  The terms of the Apache License, version 2.0 Section 3 “Grant of Patent License” do not apply to the binaries in the BitFusion Client directory which are not licensed under the Apache License, version 2.0.
```

Comment:
We should not add these claims to license file. And no Bitfusion binary should be included in this OSS project as we have discussed offline.